src/main/java/com/hydratereminder/HydrateReminderPlugin.java

```diff
@@ -232,7 +236,7 @@ public void onCommandExecuted(CommandExecuted commandExecuted)
 							handleHydrateResetCommand();
 							break;
 						case HYDRATE:
-							handleHydrateHydrateCommand();
+							commandInvoker.invokeCommand(commandExecuted);
```

Comment:
@mslowiak, @jmakhack  - temporary solution untill all commands will be refactored